src/test/java/com/alipay/remoting/rpc/RpcAddressParserTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alipay.remoting.rpc;
+
+import com.alipay.remoting.Url;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class RpcAddressParserTest {
+    private RpcAddressParser parser = new RpcAddressParser();
+
+    @Test
+    public void parseNormal() {
+        //normal ipv4 url
+        Url url = parser.parse("127.0.0.1:12200");
+        Assert.assertEquals("127.0.0.1", url.getIp());
+        Assert.assertEquals(12200, url.getPort());
+        url = parser.parse("127.0.0.1:12200?KEY1=VALUE1&KEY2=VALUE2");
+        Assert.assertEquals("127.0.0.1", url.getIp());
+        Assert.assertEquals(12200, url.getPort());
+        Assert.assertEquals(url.getProperties().size(), 2);
+
+        //normal ipv6 url
+        url = parser.parse("0:0:0:0:0:0:0:1:12200");
```

Comment:
IPv6 is compatible with many formats, and the test case here needs to be supplemented with more ipv6 addresses in different formats, for example: https://gist.github.com/syzdek/6086792

Reply:
yes，but i think it is too complex, there are many Ipv6 regex available and most of them don't work properly.  like https://rosettacode.org/wiki/Parse_an_IP_Address#Java
current support ip:port format will be ok, the same as the current ability。